### ISSUE TITLE
Mpu6050 feature

### ISSUE THREAD

**Issue by neergaby195** [MobiFlight/MobiFlight-Connector#1297]
Hello, is possible to add mpu6050 for axis because i have a modified saitek yoke and i use a mpu6050 for x axis and i discovered this project... and i like the ideea to implement.

**Comment by DocMoebiuz** [comment]
I am not sure I understand what you mean?

**Comment by tigert** [comment]
You probably should build this as USB HID, because a gyroscope / accelerometer unit will likely flood a lot of input data.

It can be read as a usb joystick axis into MobiFlight if you want to do something with the orientation data of the sensor, but for things like head tracking or controller input, it might be best to route it via the USB HID joystick driver for better performance in the sim.

**Comment by tigert** [comment]
@neergaby195 actually, if you have a look at #1265 that would be another option in the future, but custom device support is for outputs only at this point, but it could be a thing to discuss whether it makes sense for inputs too, and then it would be possible to have the MPU work as a custom input device, but this is not yet beind developed, only at ideation stage.

**Comment by JaimeLeon2** [comment]
Food for thought, I just read the MPU6050 is at end of life and it is recommended it should  not be used in any new designs.

**Comment by tigert** [comment]
Yeah, and well, there are actually now ESP32 dev boards, cheap, that contain motion and orientation sensors. These would be a nice custom firmware project for some experimental or assistive controllers, since in MobiFlight you can program and try different interaction logics as well, in addition to just airplane logic. Like, for someone with disability in their mobility, turning their head and use that as an encoder like adjustment, for example.
